js/private/npm_import.bzl

```diff
@@ -404,9 +404,9 @@ _ATTRS = {
         doc = """If true, runs lifecycle hooks declared in this package and the custom postinstall script if one exists.""",
         default = True,
     ),
-    "yq_repository": attr.string(
-        doc = """The basename for the yq toolchain repository from @aspect_bazel_lib.""",
-        default = "yq",
+    "yq": attr.string(
```

Comment:
Used a string because if label this will fail on windows.

Reply:
`attr.label` should be fine. The label here isn't evaluated in such a way to make sure it points to a real file.

Reply:
Oh cool